dll/cpl/desk/background.c

```diff
@@ -21,6 +21,12 @@
 #define PLACEMENT_FIT       3
 #define PLACEMENT_FILL      4
 
+#define PLACEMENT_VALUE_CENTER    0
+#define PLACEMENT_VALUE_STRETCH   2
+#define PLACEMENT_VALUE_TILE      0
```

Comment:
Then what the difference with PLACEMENT_VALUE_CENTER is ???

Reply:
The tile placement is stored in different registry key. I used 0 to make [this condition](https://github.com/lion328/reactos/blob/afb438af430d0f7290322cfbfe47b0ab276f4f46/win32ss/user/ntuser/sysparams.c#L693) work.

Reply:
Ah ok. Then I would probably suggest you add a small comment there, explaining this fact.